jaeger-core/src/main/java/com/uber/jaeger/Configuration.java

```diff
@@ -327,6 +372,72 @@ public String getManagerHostPort() {
     }
   }
 
+  /**
+   * CodecConfiguration can be used to support additional trace context propagation codec.
+   */
+  public static class CodecConfiguration {
+    private Map<Format<?>, List<Codec<TextMap>>> codecs;
+
+    private CodecConfiguration(Map<Format<?>, List<Codec<TextMap>>> codecs) {
+      this.codecs = codecs;
+    }
+
+    public static CodecConfiguration fromEnv() {
+      Map<Format<?>, List<Codec<TextMap>>> codecs = new HashMap<Format<?>, List<Codec<TextMap>>>();
+      String propagation = getProperty(JAEGER_PROPAGATION);
+      if (propagation != null) {
+        for (String format : Arrays.asList(propagation.split("[, ]"))) {
```

Comment:
`[, ]` means it will split on spaces too, did you intend that? I'd rather treat it only as comma-separated, the flexibility doesn't seem to be warranted.

Reply:
Was trying to be more forgiving - but will simplify